tests/app/helpers.py

```diff
@@ -89,7 +98,7 @@ class JSONUpdateTestMixin(object):
 
     def test_non_json_causes_failure(self):
         response = self.client.open(
-            self.endpoint,
+            'services/1000',
```

Comment:
What's the reason for this? `JSONUpdateTestMixin` is supposed to be a mixin class, it shouldn't have a hardcoded URL in it.